### PR TITLE
docs: add hosting guide

### PR DESCRIPTION
**Pull request by 0xRampey**
- [x] fly
- [x] ec2
- [x] railway

### CODE REVIEW

File: vocs.config.ts [hunk 1]
```diff
@@ -243,6 +243,10 @@ export default defineConfig({
               },
             ],
           },
+          {
```
Comment (aashutoshrathi) [inline vocs.config.ts:246]: should we have different items inside this as tutorials?

File: docs/pages/build/guides/hosting.mdx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+- [Fly.io CLI](https://fly.io/docs/hands-on/install-flyctl/)
+
+### Steps
+
+1. `cd` into your Micro-Rollup repository
+2. Login to `fly.io`
+```bash copy
+fly auth signup
+fly auth login // if you are already logged in
```
Comment (aashutoshrathi) [inline docs/pages/build/guides/hosting.mdx]:
```suggestion
fly auth login # if you are already logged in
```

File: docs/pages/build/guides/hosting.mdx [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+### Prerequisites
+
+- [Fly.io CLI](https://fly.io/docs/hands-on/install-flyctl/)
+
+### Steps
+
+1. `cd` into your Micro-Rollup repository
+2. Login to `fly.io`
+```bash copy
```
Comment (aashutoshrathi) [inline docs/pages/build/guides/hosting.mdx]: don't think `copy` works like this.
```suggestion
```bash [terminal]
```

File: docs/pages/build/guides/hosting.mdx [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+- Basic knowledge of AWS services
+- SSH client (for connecting to your EC2 instance)
+
+### Steps
+
+1. Follow this [guide](https://hostadvice.com/how-to/web-hosting/containers/how-to-use-docker-containers-with-aws-ec2/) to spin up an EC2 instance with Docker enabled.
+2. Build and run your rollup's docker container by following the instructions [here](/build/zero-to-one/run-it).
+
+
+## Happy rolling! 🛼
```
Comment (aashutoshrathi) [inline docs/pages/build/guides/hosting.mdx]: Is this a section?

**Review by aashutoshrathi — COMMENTED** [review]
left some comments as this is WIP

File: docs/pages/build/guides/hosting.mdx (hunk 4 above)
Comment (0xRampey) [reply docs/pages/build/guides/hosting.mdx]: no just a heading

File: vocs.config.ts (hunk 1 above)
Comment (0xRampey) [reply vocs.config.ts:246]: each guide is currently short enough to be part of the same page. Counter MRU tutorial is quite long

**Review by aashutoshrathi — APPROVED** [review]

**Pull request merged**